aws-ipam.tf

```diff
@@ -0,0 +1,46 @@
+# Create a service linked role
+resource "aws_iam_service_linked_role" "dataworks_aws_ipam_service_linked_role" { # only required for single account IPAM.
```

Comment:
Is this used by our users and is there an accompanying PR?

Reply:
No this service linked role is only for IPAM to manage the resources